odoo/addons/openupgrade_records/lib/apriori.py

```diff
@@ -7,7 +7,8 @@
     'crm_reveal': 'crm_iap_lead',
     'document': 'attachment_indexation',
     'payment_ogone': 'payment_ingenico',
-    # OCA/...
+    # OCA/hr
+    'hr_skill': 'hr_skills'
```

Comment:
This is missing the commented TODO:
```suggestion
    # TODO: Transform possible data
    'hr_skill': 'hr_skills'
```